codejl/ccopfmodel_simulation.jl

```diff
@@ -15,7 +15,8 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 # Define packages to be used:
 using JuMP # Optimization package
 using JuMPChance # Chance constraints package
-using Gurobi # Solver, could be altered if needed
+# using Gurobi # Solver, could be altered if needed
+using Mosek # Solver, could be altered if needed
```

Comment:
This change should stay local, no reason to change the solver from what we used in the paper